config/agents/svc/rbac/role.yaml

```diff
@@ -16,6 +16,25 @@ rules:
   - update
   - patch
   - delete
+- apiGroups:
+  - primaza.io
+  resources:
+  - servicebindings
+  - serviceclaims
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  verbs:
+  - get
+  - list
+  - watch
+  - update
+  - patch
```

Comment:
hum... do you need list, watch and patch? are you missing delete?

Reply:
that's the role that will be assigned to the service agent. it just need read access. I will remove update/patch.

Reply:
I still see the update and watch there